common/com.liferay.ide.ui.tests/src/com/liferay/ide/ui/tests/UIBase.java

```diff
@@ -54,7 +54,7 @@
     public final String LABEL_DELETE_RESOURCE = "Delete Resources";
     public final String LABEL_ERROR_LOG = "Error Log";
     public final String LABEL_ERROR_LOG_FULL = "Error Log (Workspace Log)";
-    public final String LABEL_LIFERAY = "Liferay Plugins";
+    public final String LABEL_LIFERAY_WORKSPACE = "Liferay Workspace";
```

Comment:
please help make sure all other test files about SDK are running correct if change to LIferay Workspace by default, thanks.